pkg/cloudprovider/servicelb.go

```diff
@@ -568,6 +568,12 @@ func (k *k3s) newDaemonSet(svc *core.Service) (*apps.DaemonSet, error) {
 					},
 				},
 			},
+			Resources: &core.ResourceRequirements{
+				Requests: &core.ResourceList{
+					"cpu": "1m",
+					"memory": "2Mi",
+				},
+			},
```

Comment:
Let's not do this. The pods just sleep after running the iptables commands and shouldn't consume a meaningful amount of resources; leaving them as BestEffort (without any requests or limits) is fine.